src/mame/namco/mappy.cpp

```diff
@@ -964,53 +965,53 @@ static INPUT_PORTS_START( phozon )
-	PORT_DIPSETTING(    0x40, DEF_STR( 2C_1C ) )
 	PORT_DIPSETTING(    0xc0, DEF_STR( 1C_1C ) )
-	PORT_DIPSETTING(    0x00, DEF_STR( 2C_3C ) )
 	PORT_DIPSETTING(    0x80, DEF_STR( 1C_2C ) )
+	PORT_DIPSETTING(    0x40, DEF_STR( 2C_1C ) )
+	PORT_DIPSETTING(    0x00, DEF_STR( 2C_3C ) )
 
-	PORT_START("DSW2")  /* 56XX #1 pins 38-41 multiplexed and interleaved */
+	PORT_START("DSW2")  // 56XX #1 pins 38-41 multiplexed and interleaved
 	PORT_DIPNAME( 0x11, 0x11, DEF_STR( Lives ) )        PORT_DIPLOCATION("SW2:2,1")
-	PORT_DIPSETTING(    0x01, "1" ) // 1 on, 2 off
 	PORT_DIPSETTING(    0x11, "3" )
 	PORT_DIPSETTING(    0x10, "4" ) // 2 on, 1 off
+	PORT_DIPSETTING(    0x01, "1" ) // 1 on, 2 off
 	PORT_DIPSETTING(    0x00, "5" )
 	PORT_DIPNAME( 0x62, 0x62, DEF_STR( Bonus_Life ) )   PORT_DIPLOCATION("SW2:4,3,5")
-	PORT_DIPSETTING(    0x02, "20k & 80k Only" )        PORT_CONDITION("DSW2",0x01,NOTEQUALS,0x00)
-	PORT_DIPSETTING(    0x40, "20k, 80k & Every 80k" )  PORT_CONDITION("DSW2",0x01,NOTEQUALS,0x00)
-	PORT_DIPSETTING(    0x20, "30k Only" )          PORT_CONDITION("DSW2",0x01,NOTEQUALS,0x00)
-	PORT_DIPSETTING(    0x42, "30k & 60k Only" )        PORT_CONDITION("DSW2",0x01,NOTEQUALS,0x00)
 	PORT_DIPSETTING(    0x62, "30k & 100k Only" )       PORT_CONDITION("DSW2",0x01,NOTEQUALS,0x00)
-//  PORT_DIPSETTING(    0x60, "30k 100k" )  // repeated         PORT_CONDITION("DSW2",0x01,NOTEQUALS,0x00)
+//  PORT_DIPSETTING(    0x60, "30k & 100k Only" )       PORT_CONDITION("DSW2",0x01,NOTEQUALS,0x00)  // repeated
+	PORT_DIPSETTING(    0x42, "30k & 60k Only" )        PORT_CONDITION("DSW2",0x01,NOTEQUALS,0x00)
+	PORT_DIPSETTING(    0x40, "20k, 80k & Every 80k" )  PORT_CONDITION("DSW2",0x01,NOTEQUALS,0x00)
 	PORT_DIPSETTING(    0x22, "30k, 120k & Every 120k" )    PORT_CONDITION("DSW2",0x01,NOTEQUALS,0x00)
 	PORT_DIPSETTING(    0x22, "20k & 80k Only" )        PORT_CONDITION("DSW2",0x01,EQUALS,0x00)
-	PORT_DIPSETTING(    0x02, "30k" )           PORT_CONDITION("DSW2",0x01,EQUALS,0x00)
-	PORT_DIPSETTING(    0x40, "30k, 100k & Every 100k" )    PORT_CONDITION("DSW2",0x01,EQUALS,0x00)
+	PORT_DIPSETTING(    0x20, "30k Only" )          PORT_CONDITION("DSW2",0x01,NOTEQUALS,0x00)
+	PORT_DIPSETTING(    0x02, "20k & 80k Only" )        PORT_CONDITION("DSW2",0x01,NOTEQUALS,0x00)
 	PORT_DIPSETTING(    0x62, "30k & 100k Only" )       PORT_CONDITION("DSW2",0x01,EQUALS,0x00)
-//  PORT_DIPSETTING(    0x60, "30k 100k" )  // repeated     PORT_CONDITION("DSW2",0x01,EQUALS,0x00)
+//  PORT_DIPSETTING(    0x60, "30k & 100k Only" )       PORT_CONDITION("DSW2",0x01,EQUALS,0x00) // repeated
 	PORT_DIPSETTING(    0x42, "40k & 80k Only" )        PORT_CONDITION("DSW2",0x01,EQUALS,0x00)
+	PORT_DIPSETTING(    0x40, "30k, 100k & Every 100k" )    PORT_CONDITION("DSW2",0x01,EQUALS,0x00)
 	PORT_DIPSETTING(    0x20, "100k Only" )         PORT_CONDITION("DSW2",0x01,EQUALS,0x00)
+	PORT_DIPSETTING(    0x02, "30k Only" )           PORT_CONDITION("DSW2",0x01,EQUALS,0x00)
```

Comment:
May as well align these if you’re editing them, too.  Also, you can uncomment the repeated conditions.  Remember the user can click on individual DIP switches now, so it’s not like they can’t produce these combinations, and it’s more useful for it to say what it does than to just say “INVALID”.